src/fides/config/execution_settings.py

```diff
@@ -61,6 +61,10 @@ class ExecutionSettings(FidesSettings):
         default=3,
         description="The number of times a privacy request will be requeued when its tasks are interrupted before being marked as error",
     )
+    async_tasks_status_polling_interval: int = Field(
```

Comment:
```suggestion
    async_tasks_status_polling_interval_seconds: int = Field(
```

I find adding the unit of time to the setting name makes it a LOT more clear as you read the code and it's a pattern we already have in other places of the codebase